code/modules/mob/living/living.dm

```diff
@@ -885,6 +875,29 @@ default behaviour is:
 /mob/living/proc/get_ingested_reagents()
 	return reagents
 
+/mob/living/proc/handle_additional_slime_effects()
+	return
+
+/mob/living/proc/slime_feed_act(var/mob/living/slime/attacker)
+	var/protection = (1 - get_blocked_ratio(null, TOX, damage_flags = DAM_DISPERSED | DAM_BIO))
+	adjustCloneLoss((attacker.is_adult ? 10 : 5) * protection)
+	adjustToxLoss(1 * protection)
+	if(health <= 0)
+		adjustToxLoss(1 * protection)
+	if(prob(15) && client)
+		handle_additional_slime_effects()
+	. = 15 * protection
+	if(stat == DEAD || getCloneLoss() >= maxHealth)
+		eaten_by_slime()
+
+/mob/living/proc/eaten_by_slime()
+	if(QDELETED(src))
+		return
+	new /obj/effect/decal/cleanable/mucus(get_turf(src))
+	dump_contents()
+	qdel(src)
+	. = rand(2,3)
```

Comment:
What does this mean?

Reply:
Feed value returned to the proc to give the slime nutrition.

Reply:
Worth a comment, as this is the proc definition and that wouldn't be one of my first 3 guesses (it's also just ignored a few lines above, I believe).

Reply:
Added a comment above this proc in the new file I moved them to.